### PR TITLE
Fixes IOException handling.

### PR DESCRIPTION
**Pull request by goober**
catching `Exception` should be avoided for bank classes and handled by the caller.

### CODE REVIEW

File: bankdroid-legacy/src/main/java/com/liato/bankdroid/banking/banks/coop/Coop.java [hunk 1]
```diff
@@ -214,17 +214,13 @@ public void update() throws BankException, LoginException, BankChoiceException,
...
-            Account a = new Account("Återbäring",
-                    Helpers.parseBalance(dResponse.select(".Heading--coopNew").text()),
-                    "refound", Account.OTHER, "SEK");
-            accounts.add(a);
-        } catch (Exception e) {
-            throw new BankException(e.getMessage(), e);
-        }
+        response = urlopen
+                .open("https://www.coop.se/Mina-sidor/Oversikt/Information-om-aterbaringen/");
+        dResponse = Jsoup.parse(response);
```
Comment (fredrike) [inline bankdroid-legacy/src/main/java/com/liato/bankdroid/banking/banks/coop/Coop.java]: We need a check if dResponse contains ".Heading--coopNew" before using it..

**Comment by goober** [comment]
ping @fredrike

File: bankdroid-legacy/src/main/java/com/liato/bankdroid/banking/banks/coop/Coop.java (hunk 1 above)
Comment (goober) [inline bankdroid-legacy/src/main/java/com/liato/bankdroid/banking/banks/coop/Coop.java]: I will do this in a separate PR.

Comment (fredrike) [inline bankdroid-legacy/src/main/java/com/liato/bankdroid/banking/banks/coop/Coop.java]: But now the code can crash uncontrollable..

**Comment by fredrike** [comment]
Thank you!

**Pull request merged**